### ISSUE TITLE
Warn for mismatched svg template tag and nodes

### ISSUE THREAD

**Issue by justinfagnani** [runem/lit-analyzer#217]
People sometimes get mixed up on when to use the `svg` template tag. A lint warning when the template tag and contents are mismatched would help. These should warn:

```ts
html`<path>...</path>`
```

```ts
svg`<svg>...</svg>`
```
